modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service.adoc

```diff
@@ -0,0 +1,120 @@
+= Connecting the Spring PetClinic sample application to the PostgreSQL database service
+[id="sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service_{context}"]
+
+For connecting the sample application to the database service, you have to create a `ServiceBinding` custom resource (CR) that triggers the {servicebinding-title} to project the binding data into the application.
+
+[discrete]
+.Procedure
+
+. Create a `ServiceBinding` CR to project the binding data:
++
+----
+$ oc apply -f - << EOD
+---
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+    name: spring-petclinic-rest
+    namespace: my-postgresql
+spec:
+    services: <1>
+    - group: postgres-operator.crunchydata.com
+      version: v1beta1
+      kind: PostgresCluster <2>
+      name: hippo
+    - group: ""
+      version: v1
+      kind: Secret
+      name: hippo-pguser-hippo
+    application: <3>
+      name: spring-petclinic-rest
+      group: apps
+      version: v1
+      resource: deployments
+----
+<1> Specifies a list of service resources.
+<2> The CR of the database.
+<3> The sample application that points to a Deployment or any other similar resource with an embedded PodSpec.
++
+The output verifies that the `ServiceBinding` CR is created to project the binding data into the sample application.
++
+.Example output
+----
+servicebinding.binding.operators.coreos.com/spring-petclinic-rest created
+----
+
+. To verify whether the binding is successful, check the status conditions of the binding resource:
```

Comment:
```suggestion
. To verify if the binding is successful, check the status conditions of the binding resource:
```
Per IBM Style Guide, "Use 'if' to express a condition and 'whether' to express an alternative." See "if versus whether conjunction" section